fedimint-client/src/lib.rs

```diff
@@ -686,7 +686,7 @@ impl Client {
     }
 
     /// Returns the config with which the client was initialized.
-    pub async fn get_config(&self) -> &ClientConfig {
+    pub fn get_config(&self) -> &ClientConfig {
```

Comment:
Probably ok to do this change, would be a problem if we ever decide to only keep the config in the DB and load on demand (unlikely imo).